src/main/java/core/basesyntax/RandoM.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class RandoM extends Enum {
+    public Color getRandomColor() {
+        Random random = new Random();
+        int index = random.nextInt(Color.values().length);
+        return Color.values()[index];
+    }
+
+    public int getRandomNumber() {
+        Random random = new Random();
+        return random.nextInt(100);
+    }
```

Comment:
Similar to the 'getRandomColor' method, reuse the 'Random' instance here instead of creating a new one each time the method is called.